spec/dummy/app/assets/templates/quuux.ja.html.slim

```diff
@@ -0,0 +1,9 @@
+doctype html
```

Comment:
@hilohiro 
Could you please use `.html.erb` (like `foo.ja.html.erb`) instead of Slim?
(Because I'd like check separately whether Slim templates can be compiled and whether localized templates can be treated correctly.)